ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -586,7 +572,6 @@ export default class ConfirmTransactionBase extends Component {
             renderSimulationFailureWarning ? null : () => this.handleEditGas()
           }
           rows={[
-            renderSimulationFailureWarning && simulationFailureWarning(),
             !renderSimulationFailureWarning && (
```

Comment:
if `renderSimulationFailureWarning` is truthy we are rendering an empty `TransactionDetail` component. You need to move the `renderSimulationFailureWarning` validation to wrap the entire TransactionDetail component.

Reply:
https://github.com/MetaMask/metamask-extension/pull/23886/commits/cb632f17edb983b176767be12805d9137dc7beea